index/scrivere-un-documento.rst

```diff
@@ -39,6 +39,8 @@ L’esempio nella :numref:`Tabella %s <cfr-rst>` illustra come è possibile indi
    |                                |                             |
    +--------------------------------+-----------------------------+
 
+Per maggiore chiarezza [in questo repository](https://github.com/italia/esempio-docsitalia) forniamo un documento di esempio contenente gli elementi di base per la documentazione. Il documento è leggibile in formato RST e nei formati usati da Office e LibreOffice, per un facile confronto.
```

Comment:
Occhio che il repository a cui fa riferimento questa documentazione https://github.com/italia/esempio-docsitalia ancora non esiste. Non so bene di cosa si tratta, ma userei un naming diverso, più coerente con altre documentazioni (`esempio-docs` @atorin aiutaci tu).

Reply:
sì infatti, prima di approvare dobbiamo decidere cosa fare di quel repo. 

Si tratta di esempi di documenti in varie versioni, incluso RST, che servono per far capire come vengono convertiti i documenti. Il nome del repo dovrebbe rispecchiare questi aspetti.

Reply:
Non sono sicuro se aggiungere TM o (R) vicino ad Office e LibreOffice? :thinking: